packages/kubernetes-1.22/Cargo.toml

```diff
@@ -14,8 +14,8 @@ path = "pkg.rs"
 package-name = "kubernetes-1.22"
 
 [[package.metadata.build-package.external-files]]
-url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/10/artifacts/kubernetes/v1.22.12/kubernetes-src.tar.gz"
-sha512 = "9856612e69c64f7420603d76c8075542104c8b00b90913f8ee04a17f6cecf3b8a187cadf91754d588fa2ed0fbc065d91a2bd7de34774bdac3f6b43240107bab1"
+url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/11/artifacts/kubernetes/v1.22.15/kubernetes-src.tar.gz"
+sha512 = "d47a5f5558b08e6e83d70520835d5f3fed7c762ec4bc1c0296ef1efbe0fd5b7babef794b68f1a5f0452a23e5d13d758dbb2cd2c93315ba8053ac607a4d2abae3"
```

Comment:
This SHA seems incorrect. I'm getting `b0bb0338ede41ed90e711fe3835b0f86a061aed19b9a2555146ea9f402246e1776ddc5f04beb6e8b74f32756726993582391827910507dc169dabc7c339b2ab2` instead. I checked the K8s version of the source and it seems correct. I'm gonna go ahead and update this checksum

Reply:
Looking deeper, it seems like the original `sha512` hash was pulled from the EKS-D release manifest https://distro.eks.amazonaws.com/kubernetes-1-22/kubernetes-1-22-eks-11.yaml for `https://distro.eks.amazonaws.com/kubernetes-1-22/releases/11/artifacts/kubernetes/v1.22.15/kubernetes-client-linux-arm64.tar.gz` and not `https://distro.eks.amazonaws.com/kubernetes-1-22/releases/11/artifacts/kubernetes/v1.22.15/kubernetes-src.tar.gz`